Web_Development/FrontEnd/HTML_Commands/HML_Commands.md

```diff
@@ -1 +1,140 @@
-hi
\ No newline at end of file
+### The html tag: 
+The <html> tag is used to specify that the text should start being interpreted as html code.
+
+### The head tag: 
+The head of an html code contains information related to the document, for example: it's title, link, style, etc. 
+
+```html 
+<html>
+   <head>
+      <title>Hello! Welcome to Girlscript Winter of Contributing</title>
+      <link rel="stylesheet" type="text/css" href="gwoc_style.css">
+   </head>
+</html>
+```
+
+### Title: 
+A title tag specifies a title in your browser window.  
+```html
+<title>Hello! Welcome to Girlscript Winter of Contributing</title>
+```
+
+### Style and Link tag:
+The style tag contains information on how html elements should render in a browser.
+```html
+<html>
+   <head>
+      <style>
+         body {
+         background: yellow;
+         }
+         h1 {
+         color: green;
+         }
+         p {
+         color: green;
+         }
+      </style>
+   </head>
+   <body>
+```
+   
+The link tag is used to link to an external style sheet.
+```html
+<html>
+   <head>
+      <link rel="stylesheet" type="text/css" href="gwoc_style.css">
+   </head>
+   <body>
+```
+      
+### The body tag: 
+The body tag specifies the beginning and end of the code that consiste of text, pictures, links, etc.
+```html
+<html>
+   <head>
+      <title>Hello! Welcome to Girlscript Winter of Contributing</title>
+   </head>
+   <body>
+```
+  
+### Headers: 
+There are up to six levels of headers that we can use in our html code, h1 to h6. Header 1 is the largest header and it decreases till header 6.
+      
+The proper way to write headings are:
+```html
+<h1>Welcome to GWOC 2021.</h1> 
+```
+      
+<h1>Welcome to GWOC 2021.</h1>
+      
+```html
+<h3>Welcome to GWOC 2021.</h3>
+```
+      
+<h3>Welcome to GWOC 2021.</h3>   
+      
+```html
+<h5>Welcome to GWOC 2021.</h5> 
+```
+      
+<h5>Welcome to GWOC 2021.</h5>     
```

Comment:
Please don't write like this, make a file in your local machine write the code there, paste the screenshot of the output here that's all.

Reply:
changed and screenshot added